pipelines/stripe_analytics/__init__.py

```diff
@@ -31,29 +22,32 @@ def stripe_source(
     stripe_secret_key: str = dlt.secrets.value,
     start_date: Optional[DateTime] = None,
     end_date: Optional[DateTime] = None,
-) -> Generator[DltResource, Any, None]:
+) -> Iterable[DltResource]:
     """
     Retrieves data from the Stripe API for the specified endpoints.
-    For all endpoints, Stripe API responses do not provide key "updated",
-    so in most cases we are forced to load the data in 'replace' mode.
-    This source is suitable for all types of endpoints, including 'Events', 'Invoice' etc.
-    but these endpoints can also be loaded in incremental mode (see source incremental_stripe_source)
+
+    For all endpoints, Stripe API responses do not provide the key "updated",
+    so in most cases, we are forced to load the data in 'replace' mode.
+    This source is suitable for all types of endpoints, including 'Events', 'Invoice', etc.
+    but these endpoints can also be loaded in incremental mode (see source incremental_stripe_source).
 
     Parameters:
```

Comment:
Let's use everywhere `Args:` to align it with Google Style

Reply:
I've marked a couple of `Parameters` occurrences, but there's a couple more in the code.